src/lib/project.ts

```diff
@@ -41,6 +41,20 @@ export async function hasRc(rootDir: string): Promise<boolean> {
   return contains;
 }
 
+/**
+ * Iterates overall package paths and, depending on the presence or absence of
```

Comment:
Document _what_ over _how_. "Determines whether the given paths describe an alle monorepo by checking for a `packages/node_modules` path in all of them."